src/finalisers/shared/getExportBlock.ts

```diff
@@ -37,18 +37,29 @@ export default function getExportBlock (
 	dependencies.forEach(({ name, reexports }) => {
 		if (reexports && exportMode !== 'default') {
 			reexports.forEach(specifier => {
-				if (specifier.imported === '*') {
+				if (specifier.reexported === '*') {
 					exportBlock += `${exportBlock ? '\n' : ''}Object.keys(${name}).forEach(function (key) { exports[key] = ${name}[key]; });`;
 				}
 			});
 		}
 	});
 
-	dependencies.forEach(({ name, reexports }) => {
+	dependencies.forEach(({ name, imports, reexports }) => {
```

Comment:
I know these two loops have previously been separate but it seems they can easily be combined into one loop which will remove one iteration + remove one if statement + make sure in a hard-coded way no reexport is handled twice.

Reply:
The reason for this separation is to ensure that the star exports come first in the output, which is important to the export ordering as star exports get overridden by reexports get overridden by exact name exports.

Reply:
Ah, I see, thanks.